### PR TITLE
Add function to search all countries

### PR DESCRIPTION
**Pull request by dk-stars**

### CODE REVIEW

File: src/main/java/com/Kh033Java/travelbook/repository/CountryRepository.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @Repository
 public interface CountryRepository extends Neo4jRepository<Country, Long> {
 	
-    List<Country> findAll();
+   // List<Country> findAll();
```
Comment (Djubabagdad) [inline src/main/java/com/Kh033Java/travelbook/repository/CountryRepository.java:15]: Can you check, please, if it still needs.

File: out/production/resources/application.properties [hunk 2]
```diff
@@ -1,12 +1,11 @@
-spring.data.neo4j.uri=bolt://35.225.138.96:7687
 
+spring.data.neo4j.uri=bolt://35.225.138.96:7687
 spring.data.neo4j.username=neo4j
-spring.data.neo4j.password=pass
+spring.data.neo4j.password=password
```
Comment (Djubabagdad) [inline out/production/resources/application.properties:4]: I think , here is no reason to change password.

File: out/production/resources/application.properties [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+
 google.cloud.bucket=travelbook
 google.cloud.credentials=src/main/resources/Bachelor-bc0312b7b74b.json
 
-jwt.token.secret=jwtappdemo
-jwt.token.expired=10000000
-
-
+token.secret=appdemo
+token.expired=3600000
```
Comment (Djubabagdad) [inline out/production/resources/application.properties:10]: There is try to rewrite current time of JWT token.

Comment (dk-stars) [reply out/production/resources/application.properties:10]: This change throw exception

**Pull request closed without merging**